snowpack/src/sources/local.ts

```diff
@@ -493,9 +493,15 @@ export class PackageSourceLocal implements PackageSource {
       const lineBullet = colors.dim(depth === 0 ? '+' : '└──'.padStart(depth * 2 + 1, ' '));
       let packageFormatted = spec + colors.dim('@' + packageVersion);
       const existingImportMapLoc = path.join(installDest, 'import-map.json');
-      const existingImportMap: ImportMap | null =
-        (await fs.stat(existingImportMapLoc).catch(() => null)) &&
-        JSON.parse(await fs.readFile(existingImportMapLoc, 'utf8'));
+      const importMapHandle = await fs.open(existingImportMapLoc, 'r+').catch(() => null);
```

Comment:
Like the PR notes said, it’s hard for me to test/confirm this but overall this seems like a pretty safe change to me, so I’m +1